spec/controllers/reviews_controller_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'spec_helper'
+
+describe ReviewsController do
+  describe "POST create" do
+
+    context "valid input" do
+      let(:current_user) { Fabricate(:user) }
+      before { session[:user_id] = current_user.id }
+      let(:video) { Fabricate(:video) }
+      
+      it "sets @video" do
+        post :create, video_id: video.id, review: Fabricate.attributes_for(:review)
+        expect(assigns(:video)).to eq(video)
+      end
+      it "sets @review" do  
```

Comment:
the `"sets @..."` means that it is actually setting the variable in the controller. This test isn't about that so I would rename it.